charts/incubator/tt-rss/questions.yaml

```diff
@@ -9,22 +9,48 @@ questions:
 # Include{replicas1}
 # Include{podSpec}
 # Include{containerMain}
-                                - variable: env
-                                  group: App Configuration
-                                  label: Image Environment
-                                  schema:
-                                    additional_attrs: true
-                                    type: dict
-                                    attrs:
-                                      - variable: TTRSS_SELF_URL_PATH
-                                        label: TTRSS Self URL Path
-                                        description: Sets the URL for the app.
-                                        schema:
-                                          type: string
-                                          default: ""
 # Include{containerBasic}
 # Include{containerAdvanced}
 # Include{containerConfig}
+  - variable: env
+    group: App Configuration
+    label: Image Environment
+    schema:
```

Comment:
The indentation is off! It needs to be a lot more indented (check other apps from stable!)